front/app/containers/ProjectsShowPage/shared/document_annotation/Konveio.test.tsx

```diff
@@ -1,30 +1,24 @@
 import React from 'react';
 import { render, screen } from 'utils/testUtils/rtl';
 
-import KonveioSurvey from './KonveioSurvey';
+import Konveio from './Konveio';
 
-describe('KonveioSurvey', () => {
+describe('Konveio', () => {
   it('should render', () => {
     render(
-      <KonveioSurvey
-        email={null}
-        konveioSurveyUrl={'https://demo.konveio.com/node/93?iframe=true'}
-      />
+      <Konveio documentUrl={'https://demo.konveio.com/node/93?iframe=true'} />
     );
     expect(screen.getByTestId('konveiosurvey')).toBeInTheDocument();
   });
 
   it('append email on query', () => {
```

Comment:
Do we still need this test? If so, we could change it's name

Reply:
Yes, and I'm not going to change the name but fix the test. The mock data was missing :-)

Reply:
Also changed the name*